src/AmplNLWriter.jl

```diff
@@ -71,7 +72,7 @@ function call_solver(
         # the BLAS library via the LBT_DEFAULT_LIBS environment variable.
         # Provide a default in case the user doesn't set.
         lbt_default_libs = get(ENV, "LBT_DEFAULT_LIBS", _get_blas_loaded_libs())
-        cmd = `$(solver_path) $(nl_filename) -AMPL $(options)`
+        cmd = `$solver_path $nl_filename $(solver.flags) $options`
```

Comment:
I want to revert all of these changes once https://github.com/cvanaret/Uno/issues/31 is fixed